src/env-inl.h

```diff
@@ -256,8 +256,13 @@ inline void AsyncHooks::RemoveContext(v8::Local<v8::Context> ctx) {
   v8::Isolate* isolate = env()->isolate();
   v8::HandleScope handle_scope(isolate);
   for (auto it = contexts_.begin(); it != contexts_.end(); it++) {
+    if (it->IsEmpty()) {
+      contexts_.erase(it);
+      it--;
```

Comment:
```suggestion
      it = contexts_.erase(it);
      it--;
```

since technically the `erase()` call turns `it` into an invalid/unusable iterator?